src/pages/config/config-changes.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import { useOutletContext } from "react-router-dom";
+
+import { getAllChanges } from "../../api/services/configs";
+
+import { ConfigChangeHistory } from "../../components/ConfigChangeHistory";
+import { toastError } from "../../components/Toast/toast";
+import { BreadcrumbNav } from "../../components/BreadcrumbNav";
+
+export function ConfigChangesPage() {
+  const [data, setData] = useState([]);
+  const [isLoading, setLoading] = useState(true);
+  const { setTitle } = useOutletContext();
+
+  useState(() => {
+    setLoading(true);
+    getAllChanges()
+      .then((res) => {
+        setData(res?.data?.length === 0 ? [] : res?.data);
+        setTitle(<BreadcrumbNav list={["Config"]} />);
```

Comment:
inline literals introduces unnecessary react dom reconciliations. 
Can we use memoized constants here.